packages/medusa/src/services/sales-channel-inventory.ts

```diff
@@ -0,0 +1,48 @@
+import { EntityManager } from "typeorm"
+
+import { IInventoryService } from "../interfaces/services"
+
+import { SalesChannelLocationService, EventBusService } from "./"
+
+type InjectedDependencies = {
+  inventoryService: IInventoryService
+  salesChannelLocationService: SalesChannelLocationService
+  eventBusService: EventBusService
+  manager: EntityManager
+}
+
+class SalesChannelInventoryService {
+  protected manager: EntityManager
+
+  protected readonly salesChannelLocationService: SalesChannelLocationService
+  protected readonly eventBusService: EventBusService
+  protected readonly inventoryService: IInventoryService
```

Comment:
todo: Should be suffixed with `_`
```suggestion
  protected readonly salesChannelLocationService_: SalesChannelLocationService
  protected readonly eventBusService_: EventBusService
  protected readonly inventoryService_: IInventoryService
```